README.md

```diff
@@ -18,7 +18,53 @@ In fact, to run tests or basically anything with Django management commands, you
 
 ### Environmental Settings
 
-A sample development environment settings file is in `.environ/.env.dev`. You can copy this over to your root as `.env` to use if you'd like to run `make tests` outside of a Docker container, for example.
+A sample development environment settings file is in `.environ/.env.dev`. You can copy this over to your root as `.env` to use if you'd like to run `make test` outside of a Docker container, for example.
+
+### JIRA Worklogs Integration
+
+Invoices can be loaded with line items derived from JIRA Tempo worklogs, given that the username of the provider on the invoice is equal to a username on a JIRA instance.
+
+Synchronization is guaranteed -- any line item that's tagged as a JIRA worklog will be checked against incoming JIRA worklogs, and a mismatch leads to deletion, whereas ones that don't exist yet always get added, thus ensuring a complete match for all relevant fields -- issue title, worklog description, worklog quantity, and so on.
```

Comment:
Deleted, actually. I'm not sure if `simple_history` will store these changes for the invoice model or not already though, i.e. "this line item was deleted for the invoice".

Reply:
Yup, I saw how this was handled below. It might be worth testing a bit what happens - from the doc alone, it's not clear what happens when referenced objects are deleted?

Reply:
@UmanShahzad Do you have an answer here?

Reply:
@antoviaque Oh, I forgot (sorry, there's so many comments here): I have not tested this out explicitly, no. I'll take a look at the state of the invoice's history in the DB when I delete a line item from an invoice, and get back to you in a bit.

Reply:
@antoviaque Actually to keep with our new model of working, since it's so late into the sprint, I'm going to make a new ticket for this, to not only investigate but also discuss the ramifications if it does something against what we want, and propose a solution.